api-specs/api_spec_PGS.yaml

```diff
@@ -575,6 +575,41 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/XPayRefundResponse500'
+  /request-payments/creditCard/{requestId}:
+    get:
+      summary: retrieve CreditCard payment request
+      tags:
+        - Credit Card
+      parameters:
+        - in: path
+          required: true
+          name: requestId
+          description: Id of the request
+          example: "41bc2409-5926-4aa9-afcc-797c7054e467"
+          schema:
+            type: string
+        - in: header
+          name: MDC-Fields
+          description: MDC information
+          example: 97g10t83x7bb0437bbc50sdf58e970gt
+          schema:
+            type: string
+          required: false
+      responses:
+        "200":
+          description: OK
+          content:
+            '*/*':
```

Comment:
Unless:
 1) CreditCardPaymentController can return different content types via the `Accept` header, or
 2) This 200 response can actually map to any content type, or
 3) There is another reason to leave the spec without an explicit content type

I would specify the content type here. :)

Reply:
Since this swagger is only used by Simone to create the endpoints on the APIM, we never really specified the content type and maybe other specific informations. But if you want I can still it, let me know.